README.md

```diff
@@ -40,90 +40,33 @@ An easy way to get the full Panoptes stack running (see `fig.yml` to dig into th
   * [Ubuntu](https://docs.docker.com/installation/ubuntulinux/) - Docker
   * [Windows](http://docs.docker.com/installation/windows/) - Boot2Docker
 
-#### Installation
+* [fig](http://fig.sh)
 
-1. Ensure your repo directory starts with a lowercase letter - you may need to move `/Panoptes` to `/panoptes`.
+#### Usage
 
-2. Prepare your fig development environment config files. You should only have to do this before the first boot. **Note:** The fig docker environment uses linked docker containers, so your Postgres and Zookeeper hosts urls need to refer to these containers.
+1. Setup the application configuration files by running `rake configure FIG_RAKE=off`
```

Comment:
By this I meant reversing the behaviour of the gem for all monkey patched commands, a la [here](https://github.com/edpaget/fig_rake/blob/0387991954ff88db057041812b79157c1258b24a/lib/fig_rake/rails.rb#L31)

``` ruby
define_method method do
  if ENV['FIG_RAKE'] == 'on'
    run_in_fig(argv.unshift(method))
  else
    m.bind(self).call
  end
end
```